README.md

```diff
@@ -191,9 +191,9 @@ All transported data is formed with a header followed by an optional payload. Th
       <b>Frame Type = 0x00 (Control Frame)</b><br>
       0x0 - 0xFFFFFFFF reserved.<br>
       <b>Frame Type = 0x02 (First Frame)</b><br>
-      0x08 The data size for a first frame is always 8 bytes. In the payload, the first four bytes denote the Total Size of the data contained in all consecutive frames, and the second four bytes denote the number of consecutive frames following this one<br>
```

Comment:
> 0x08 The data size for a first frame is always 8 bytes.

Any reason for removing this note?

Reply:
@jacobkeeler looks like it was removed by mistake. I restored the first sentence here. https://github.com/smartdevicelink/protocol_spec/pull/41/commits/14c425acd8be165cc36d355bfb658c98b336240a